pkg/agent/flowexporter/flowrecords/flow_records.go

```diff
@@ -109,6 +110,19 @@ func (fr *FlowRecords) GetFlowRecordFromMap(connKey *flowexporter.ConnectionKey)
 	return &record, exists
 }
 
+// GetFlowRecordFromMap gets the flow record from record map given connection key.
+// This is used only for unit tests.
```

Comment:
This comment does not applicable here. We should also remove `This is used only for unit tests.` comment from `GetFlowRecordFromMap`